tutorial/index.jade

```diff
@@ -136,13 +139,13 @@
       In order to end up with a cleaner and decoupled design we'll encapsulate all Authy's related features in a `AuthyService`.
       This class holds a shared class instance of the `AuthyApiClient` class.
 
-      Once the user has an `authyId`, we can actually send a verification code
-      to that user's mobile phone. 
+      Once the user has an `authyId` we can actually send a verification code
+      to that user's mobile phone.
 
-      When our user is created successfully via the form we implemented, we send
+      When our user is created successfully, via the form we implemented, we send
       a token to the user's mobile phone to verify their account in our
-      controller. When the code is sent, we redirect to another page where the
-      user can enter the token they received, completing the verification process.
+      controller. When the code is sent, we redirect him to another page where
```

Comment:
Not a problem or anything but we usually use the [singular, gender-neutral they](http://www.americandialect.org/2015-word-of-the-year-is-singular-they) or default to she/her.